runtime/binding-http/src/test/java/io/aklivity/zilla/runtime/binding/http/internal/streams/rfc7230/server/AccessControlIT.java

```diff
@@ -340,4 +340,15 @@ public void shouldAllowCredentialsCookie() throws Exception
     {
         k3po.finish();
     }
+
+    @Test
+    @Configuration("server.access.control.cross.origin.allow.explicit.yaml")
+    @Specification({
+        "${net}/ignore.explicit.allow.origin/client",
+        "${app}/ignore.explicit.allow.origin/server",
+    })
+    public void shouldIgnoreExplicitAllowOrigin() throws Exception
+    {
+        k3po.finish();
+    }
```

Comment:
Looks good.

We also need `shouldIgnoreAllowOrigin()` that uses `server.access.control.cross.origin.yaml` and confirms that `*` is not sent, otherwise we're not yet proving which value is taking precedence with only `shouldIgnoreExplicitAllowOrigin()`.